.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
- [ ] add `.idea` to .gitignore so that all the .idea files do not show up in the repo